src/services/parserContext.services.ts

```diff
@@ -48,29 +48,33 @@ spdxJsonParser = function (sbom): Package[] {
   sbomPackages.forEach(function (pkg: any) {
     let p: Package = {
       name: pkg.name,
-      id: pkg.SPDXID,
-      purl: undefined,
-      cpeName: undefined,
-      impact: undefined,
-      consRisk: undefined,
-      highestRisk: undefined,
-      likelihood: undefined,
-      version: pkg.versionInfo,
+      ref: pkg.SPDXID,
+      purl: null,
+      cpeName: null,
+      impact: null,
+      consRisk: null,
+      highestRisk: null,
+      likelihood: null,
+      version: null,
     };
 
     //Get External Reference locators
```

Comment:
explain here that a for loop is used since multiple external references are found and we're opting to take either purl or cpe